### PR TITLE
Add a codeowners file to repo

### PR DESCRIPTION
**Pull request by wagnst**
GitHub `CODEOWNERS` file to be present in repo so the setting in this repo can be changed and a good default in reviewers is set.

### CODE REVIEW

**Comment by danielmohacsi** [comment]
Dear @wagnst ,

This PR is violating the [contributing guidelines](https://github.com/lhsystems/technology-radar/blob/master/CONTRIBUTING.md) of the repository.

Please add a meaningful description for the Pull Request itself in the Description section.

PS: I would also like to raise your attention to the fact, that you are not following the commit message format either, your commit message is not in present term. Please watch out for this in your next contribution!

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# This is a comment.
+# Each line is a file pattern followed by one or more owners.
+
+# These owners will be the default owners for everything in
+# the repo. Unless a later match takes precedence,
+# @global-owner1 and @global-owner2 will be requested for
+# review when someone opens a pull request.
+*       @technology-coe
```
Comment (danielmohacsi) [inline .github/CODEOWNERS:8]: Please add a newline character at the end of the file.
For details, check the contributing guidelines, and the [POSIX definition of a line](https://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap03.html#tag_03_206).

**Review by danielmohacsi — CHANGES_REQUESTED** [review]
Missing newline at EOF. This is not standard, even if it works.

File: .github/CODEOWNERS (hunk 1 above)
Comment (wagnst) [reply .github/CODEOWNERS]: thanks! its added in another commit!

**Review by danielmohacsi — APPROVED** [review]
LGTM

**Pull request merged**